### PR TITLE
Sound Environment work

### PR DESCRIPTION
**Pull request by JakeDaBoss**
This update aims to add onto existing systems by differentiating room sound environments and how sounds reflect and are perceived inside these spaces.

### CODE REVIEW

File: maps/DeadSpace/Aegis_areas.dm [hunk 1]
```diff
@@ -4,41 +4,53 @@
 	name = "\improper Aegis VII"
 	icon = 'maps/DeadSpace/ishimura_areas.dmi'
 	icon_state = "aegis"
+	sound_env = ASTEROID
```
Comment (NanakoAC) [inline maps/DeadSpace/Aegis_areas.dm:7]: While we've not gotten around to fixing this yed, aegis VII is not an asteroid. It's not meant to be the lightless, airless rock it currently is. Aegis VII in canon is a rocky desert planet with a breathable atmosphere and a harsh noonday sun

**Review by NanakoAC — COMMENTED** [review]
small comment about aegis

File: maps/DeadSpace/Aegis_areas.dm (hunk 1 above)
Comment (JakeDaBoss) [reply maps/DeadSpace/Aegis_areas.dm:7]: This is for a sound environment. It's for the reverb and sound reflection to take effect.

This is simply a very, very large open area basically. So, the issue you see is a non-issue.

**Comment by JakeDaBoss** [comment]
Remind me to kick @Snypehunter007's ass for not doing this to begin with, big rees!

**Pull request merged**